frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -17,6 +20,45 @@ if Device:hasFrontlight() then
     }
 end
 
+local function setTime(hour, min)
```

Comment:
I wonder if this should go into Device. A `Device:getTime()` and `Device:setTime()` kind of make sense to me. Then you could do `if Device:setTime() then` instead of the isKobo or isKindle or etc.

What do you think @houqp @Hzj-jie ?